### PR TITLE
Update main

### PR DESCRIPTION
**Pull request by Mohan29997**

### CODE REVIEW

File: packages/data/src/out/network_port.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { MyFirstModel } from 'packages/shared/src/shared'
+import { FirstModel } from 'shared'
 
 export interface NetworkPort {
-  yourFirstNetworkCall(): Promise<MyFirstModel>
+  login(params?: { email: string; password: string }): Promise<FirstModel>
```
Comment (faiyyazs) [inline packages/data/src/out/network_port.ts:4]: make it firtApi

File: packages/database-watermelon/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "author": "Mohan29997 <kotakmohan33@gmail.com>",
   "homepage": "https://github.com/NeoSOFT-Technologies/mobile-react-native#readme",
   "license": "ISC",
-  "main": "lib/database-watermelon.js",
+  "main": "src/index.js",
```
Comment (faiyyazs) [inline packages/database-watermelon/package.json:8]: check this path as we use ts

File: packages/domain/src/di/domain_module.ts [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+import { FirstRepository } from '../repository/first_repository'
+import { YourFirstUseCase } from '../usecases/your_first_usecase'
 
 @Singleton()
 @Graph({ subgraphs: [DataModule] })
 export class DomainModule extends ObjectGraph {
   @Provides()
-  providesYourFirstUseCase(repo: FirstRepository): YourFirstUseCase {
-    return new YourFirstUseCase(repo)
+  providesYourFirstUseCase(provideUserRepository: FirstRepository): YourFirstUseCase {
```
Comment (faiyyazs) [inline packages/domain/src/di/domain_module.ts:10]: rename

File: packages/domain/tsconfig.json [hunk 4]
```diff
@@ -3,5 +3,11 @@
   "compilerOptions": {
     "outDir": "./lib"
   },
-  "include": ["./src"]
+  "include": [
+    "./src",
+    "../mobile/src/di/appIntilizer.ts",
```
Comment (faiyyazs) [inline packages/domain/tsconfig.json:8]: remove

File: packages/localisation/lib/en.json [hunk 5]
```diff
@@ -1,4 +1,16 @@
 {
   "appName": "MY APP",
-  "anotherString": "Hello"
+  "anotherString": "Hello",
```
Comment (faiyyazs) [inline packages/localisation/lib/en.json:3]: keep string which are used

File: packages/mobile/ios/Podfile [hunk 6]
```diff
@@ -18,7 +18,7 @@ target 'mobile' do
     # Hermes is now enabled by default. Disable by setting this flag to false.
     # Upcoming versions of React Native may rely on get_default_flags(), but
     # we make it explicit here to aid in the React Native upgrade process.
-    :hermes_enabled => true,
+    :hermes_enabled => false,
```
Comment (faiyyazs) [inline packages/mobile/ios/Podfile:21]: enable it please

**Review by faiyyazs — APPROVED** [review]

File: packages/domain/src/di/domain_module.ts (hunk 3 above)
Comment (Mohan29997) [reply packages/domain/src/di/domain_module.ts]: done

File: packages/localisation/lib/hi.json [hunk 7]
```diff
@@ -1,4 +1,13 @@
 {
   "appName": "मेरा एपीपी",
-  "anotherString": "नमस्ते"
+  "anotherString": "नमस्ते",
```
[the remark this answers is not in the document]
Comment (Mohan29997) [reply packages/localisation/lib/hi.json]: done

**Pull request merged**